File: presentation/src/main/java/com/bunbeauty/presentation/item/MenuProductItem.kt
package com.bunbeauty.presentation.item

import android.graphics.drawable.Drawable
import com.bunbeauty.domain.model.BaseItem
import java.lang.ref.SoftReference

data class MenuProductItem(
    override var uuid: String,
    val name: String,
    val newPrice: String,
    val oldPrice: String?,
    val photoLink: String,
    var photoNotWeak: SoftReference<Drawable?> = SoftReference(null)
) : BaseItem()

File: PapaKarloSwift/PapaKarloSwift/feature/menu/MenuProductItem.swift
//
//  MenuProductItem.swift
//  PapaKarloSwift
//
//  Created by Марк Шавловский on 10.03.2022.
//

import SwiftUI
struct MenuProductItem:Identifiable  {
     let id: UUID
     let name: String
     let newPrice: String
     let oldPrice: String?
     let photoLink: String
}
